app/src/main/java/com/mupper/gobus/commons/extension/Context.kt

```diff
@@ -0,0 +1,23 @@
+package com.mupper.gobus.commons.extension
```

Comment:
Great! I think I'm going to split my extensions too :P

Reply:
I see that you implement your data mappers similarly.